tasks/metrics/mutant.rake

```diff
@@ -19,6 +19,13 @@ namespace :metrics do
       %W[--ignore #{matcher}]
     end
 
+    jobs =
```

Comment:
Since the conditional branches are so small you could probably write this as a one-liner:

```ruby
jobs = ENV.key?('CIRCLECI') ? %w[--jobs 4] : []
```